src/tribol/geom/GeomUtilities.cpp

```diff
@@ -1474,6 +1470,26 @@ void PolyReorder( real* const RESTRICT x, real* const RESTRICT y, const int numP
 
 } // end PolyReorder()
 
+//------------------------------------------------------------------------------
+void PolyReverse( real* const RESTRICT x, real* const RESTRICT y, const int numPoints )
+{
+   real xtemp[ numPoints ];
```

Comment:
Warning: Runtime-sized arrays are non-standard and might fail w/ certain compilers and/or flags.

This operation can be done in place using something like this: 
https://github.com/LLNL/axom/blob/c35431a1ee71a7c46fe54805e8aa5278120d613e/src/axom/primal/geometry/BezierCurve.hpp#L296-L301

Reply:
I will make an issue. This is outside the scope of this PR, but does need to be addressed.